fix: fixes accent colors for blockquote and headings in dark mode

andrew-boyd opened the pull request:
Dark mode has incorrect accents. This PR updates the stylesheet to create light-color accents when in dark mode for headings and blockquotes.

**before**
![Screenshot 2025-03-18 at 10 11 06 AM](https://github.com/user-attachments/assets/0dcd61f9-56a7-48d7-a07b-95ba0924398b)

**after**
![Screenshot 2025-03-18 at 10 17 56 AM](https://github.com/user-attachments/assets/165b3a4b-f739-4a4a-ba92-bcdf34a29e67)

DigitallyTailored commented:
Ooh this is an interesting one.. I originally had it setup so that we only had a light mode so the `-dark` var was just for an inverted colour within it. It might be better to change the variable name from `dark` to `invert` as that's a bit clearer I think. 

I kind of liked the black bars in the dark theme originally.. Let me think about that too!

andrew-boyd commented:
Understood, this one is totally subjective.

DigitallyTailored commented:
if you wanted to change the variable names to invert I'll be sure to merge it so you can get a PR in! Maybe 

```
    --primary-highlight
    --primary-invert
```

What do you reckon?

andrew-boyd commented:
@DigitallyTailored I changed the variable names and also fixed an issue where the button classes were being overridden by the dark mode specificity. They should all retain their proper colors now.

![Screenshot 2025-03-19 at 1 22 08 PM](https://github.com/user-attachments/assets/dc5f80e9-ae28-4ad0-9473-184a75570abe)

DigitallyTailored commented:
I love it! There are a couple of conflicts - I changed a couple of vars too. I'll resolve these and merge this shortly.
